### DISCUSSION TITLE
como parar de rodar o servidor

### DISCUSSION THREAD

**Discussion by MarcioFreitasUFSC21** [balta-io/forum#304] [Tira Dúvidas] [answered]
Boa tarde, professor!
professor, como a gente faz para sair do nodemon quando ele está rodando o servidor numa porta qualquer para que não ter que desligar o terminal a todo momento?
há uma tecla do vsCode para isso?
fico no aguardo.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @MarcioFreitasUFSC21, como vai?

O atalho que para a execução é `CTRL+C`.
Abraços

**Reply by MarcioFreitasUFSC21** [reply]
obrigado, professor!funcionou aqui!valeu!
